include/nuttx/video/rgbcolors.h

```diff
@@ -33,10 +33,9 @@
 
 /* Color Creation and Conversion Macros *************************************/
 
-/* This macro creates RGB24 from 8:8:8 RGB */
-
 #define RGBTO24(r,g,b) \
-  ((uint32_t)((r) & 0xff) << 16 | (uint32_t)((g) & 0xff) << 8 | (uint32_t)((b) & 0xff))
+  ((uint32_t)((r) & 0xff) << 16 | (uint32_t)((g) & 0xff) << 8 | \
```

Comment:
why change

Reply:
line length - which I think applies to .h files? Can't remember if checkpatch complained or if I just do this automatically now!

Reply:
reverted